src/ngrest/plugins/SelectArrayGently.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace luya\admin\ngrest\plugins;
+
+use luya\admin\helpers\Angular;
+use luya\admin\traits\LazyDataLoadTrait;
+use luya\helpers\ArrayHelper;
+
+/**
+ * Nondestructive analogue of SelectArray plugin.
+ *
+ * Create a selection based on an assoc array provided via $data attribute.
+ * Will NOT override the default values from the database.
+ *
+ *
+ * Example usage:
+ *
+ * ```php
+ * public function ngRestAttributeTypes()
+ * {
+ *     'genres' => ['selectArrayGently', 'data' => [1 => 'Male', 2 => 'Female']],
+ * }
+ * ```
+ * Or use a closure for lazy data load:
+ *
+ * ```php
+ * public function ngRestAttributeTypes()
+ * {
+ *     return [
+ *         'genres' => ['selectArrayGently', 'data' => function () {
+ *               return new Query()->all();
+ *          }],
+ *     ];
+ * }
+ * ```
+ *
+ * @property array $data Setter/Getter for the dropdown values.
+ *
+ * @author Anton Ikonnikov <antikon2@yandex.ru>
+ * @since 4.1.0
+ */
+class SelectArrayGently extends Select
+{
+    use LazyDataLoadTrait;
+    
+    private $_data;
+
+
```

Comment:
please remove the empty break and add @inheritdoc

Reply:
added some comments (@inheritdoc refers to nowhere for $_data)